develop/parachains/maintenance/unlock-parachain.md

```diff
@@ -0,0 +1,131 @@
+
+2. Build the `registrar.removeLock` extrinsic
+    1. Select the **registrar** pallet
+    2. Choose the **removeLock** extrinsic
+    3. Fill in the parachain ID parameter (e.g. `2006`)
+    4. Copy the encoded call data
+
+    ![](/images/develop/parachains/maintenance/unlock-parachain/unlock-parachain-2.webp)
+
+3. Determine the transaction weight required for executing the call. You can estimate this by executing the `transactionPaymentCallApi.queryCallInfo` runtime call with the encoded call data previously obtained:
+
+    ![](/images/develop/parachains/deployment/coretime-renewal/coretime-renewal-3.webp)
+
+    This weight information is crucial for properly configuring your XCM message's execution parameters in the next steps.
+
+### Fund the Sovereign Account
+
+For a successful XCM execution, the [sovereign account](https://github.com/polkadot-fellows/xcm-format/blob/10726875bd3016c5e528c85ed6e82415e4b847d7/README.md?plain=1#L50){target=\_blank} of your parachain on the relay chain must have sufficient funds to cover transaction fees. The sovereign account is a deterministic address derived from your parachain ID.
+
+You can identify your parachain's sovereign account using either of these methods:
+
+- Use the **"Para ID" to Address** section in [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/){target=\_blank} with the **Sibling** option selected
+
+- Calculate it manually:
+
+    1. Identify the appropriate prefix:
+
+        - For parent/child chains use the prefix `0x70617261` (which decodes to `b"para"`)
+         
+    2. Encode your parachain ID as a u32 [SCALE](https://docs.polkadot.com/polkadot-protocol/basics/data-encoding/#data-types){target=\_blank} value:
+
+        - For parachain 2006, this would be `d6070000`
+
+    3. Combine the prefix with the encoded ID to form the sovereign account address:
+
+        - **Hex** - `0x70617261d6070000000000000000000000000000000000000000000000000000`
+        - **SS58 format** - `5Ec4AhPW97z4ZyYkd3mYkJrSeZWcwVv4wiANES2QrJi1x17F`
+  
+You can transfer funds to this account from any account on the relay chain using a standard transfer.
+
+### Craft and Submit the XCM
+
+With the call data prepared and the sovereign account funded, you can now construct and send the XCM from your parachain to the relay chain. The XCM will need to perform several operations in sequence:
+
+1. Withdraw DOT from your parachain's sovereign account
+2. Buy execution to pay for transaction fees
+3. Execute the `registrar.removeLock` extrinsic
+4. Return any unused funds to your sovereign account
+
+Here's how to submit this XCM using Astar (Parachain 2006) as an example:
+
+1. In [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank}, connect to the parachain, navigate to the **Developer** dropdown and select the **Extrinsics** option
```

Comment:
it would be good to provide a link with an example call on pjs like [this one](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.rpc.subquery.network%2Fpublic%2Fws#/extrinsics/decode/0x4602ea070000) and to tell which fields to tweak